lib/dish.rb

```diff
@@ -0,0 +1,28 @@
+class Dish
+
+
+	def initialize(name, price)
+			@name = name
+			@price = price
+	end
+
+	# getters
+	def name
+		@name
+	end
+
+	def price
+		@price
+	end
+
+	# setters
+	def dish(name)
+		@name = name
+	end
+
+	def dish(price)
+		@price = price
+	end
+
```

Comment:
These two methods have the same name - remember that the argument in the parentheses is just like a placeholder and does not change the name of the method.